### PR TITLE
fix: add xterm and rio color profiles

### PR DESCRIPTION
**Pull request by aymanbagabas**
This adds `TERM=rio` as a TrueColor profile and `TERM=xterm` as an ANSI profile.

### CODE REVIEW

**Review by caarlos0 — APPROVED** [review]
noice

**Comment by meowgorithm** [comment]
TERM=@raphamorim 👀

**Review by meowgorithm — APPROVED** [review]

**Comment by raphamorim** [comment]
let's goooo
image

**Comment by caarlos0** [comment]
honestly liked it better checking whole terminal names instead of splitting by `-` 🤔

**Comment by aymanbagabas** [comment]
> honestly liked it better checking whole terminal names instead of splitting by `-` 🤔

Some terminals like `st` have different variants, such as `st-256color` and `st-meta`. There's also `xterm-direct` that we should add, which also supports true colors.

**Comment by caarlos0** [comment]
@aymanbagabas i would prob just add all of them statically 🤔

**Comment by aymanbagabas** [comment]
> @aymanbagabas i would prob just add all of them statically 🤔

@caarlos0 This PR build on this one and adds support for Terminfo and queries Tmux https://github.com/charmbracelet/colorprofile/pull/24

File: env.go [hunk 1]
```diff
@@ -119,12 +119,30 @@ func colorTerm(env map[string]string) bool {
...
+		"st",
+		"wezterm":
+		return TrueColor
+	case "xterm":
+		for _, t := range []string{
+			// These terminals can be defined as xterm-TERMNAME
+			"ghostty",
+			"kitty",
+		} {
+			if strings.Contains(term, "-"+t) {
```
Comment (caarlos0) [inline env.go:141]: maybe strings.HasSuffix?

File: env.go [hunk 2]
```diff
@@ -119,12 +119,30 @@ func colorTerm(env map[string]string) bool {
...
+		"contour",
+		"foot",
+		"ghostty",
+		"kitty",
+		"rio",
+		"st",
+		"wezterm":
+		return TrueColor
+	case "xterm":
+		for _, t := range []string{
```
Comment (caarlos0) [inline env.go:136]: i wonder if this would be more readable as 

```go
switch parts[1] {
case "ghostty", "kitty":
  return TrueColor
}
```

**Review by caarlos0 — APPROVED** [review]

File: env.go (hunk 1 above)
Comment (aymanbagabas) [reply env.go:141]: Handled these in https://github.com/charmbracelet/colorprofile/pull/24

File: env.go (hunk 2 above)
Comment (aymanbagabas) [reply env.go:136]: Same

**Pull request merged**